binr/radare2/radare2.c

```diff
@@ -527,6 +527,22 @@ int main(int argc, char **argv, char **envp) {
 		return 0;
 	}
 
+	// Try to set the correct scr.color for the current terminal.
+	if ((tmp = r_sys_getenv ("COLORTERM")) && (r_str_endswith (tmp, "truecolor") || r_str_endswith (tmp, "24bit"))) {
```

Comment:
What happens if the user specifies a different value in radare2rc or the -e flags?

Reply:
from my tests, it works as you'd expect. radare2rc/-e have precedence over this automagic setting.

Reply:
yeah, that was the idea behind it. something that selects a sane default for the current console.